src/templates/static/get_started/cfds.jsx

```diff
@@ -113,14 +112,14 @@ const Cfds = () => (
                                 { text: it.L('Volume step'),    className: 'gr-padding-10 w-80' },
                             ]],
                             tbody: [
-                                [{ text: 'Volatility 10 Index'     }, { text: it.L('USD 1 per point') }, { text: tenth       }, { text: hundredth }],
-                                [{ text: 'Volatility 25 Index'     }, { text: it.L('USD 1 per point') }, { text: five_tenth  }, { text: hundredth }],
-                                [{ text: 'Volatility 50 Index'     }, { text: it.L('USD 1 per point') }, { text: five_tenth  }, { text: hundredth }],
-                                [{ text: 'Volatility 75 Index'     }, { text: it.L('USD 1 per point') }, { text: hundredth   }, { text: hundredth }],
-                                [{ text: 'Volatility 100 Index'    }, { text: it.L('USD 1 per point') }, { text: hundredth   }, { text: hundredth }],
-                                [{ text: 'HF Volatility 10 Index'  }, { text: it.L('USD 1 per point') }, { text: tenth       }, { text: hundredth }],
-                                [{ text: 'HF Volatility 50 Index'  }, { text: it.L('USD 1 per point') }, { text: five_tenth  }, { text: hundredth }],
-                                [{ text: 'HF Volatility 100 Index' }, { text: it.L('USD 1 per point') }, { text: three_tenth }, { text: hundredth }],
+                                [{ text: 'Volatility 10 Index'     }, { text: it.L('USD 1 per point') }, { text: five_tenth }, { text: hundredth }],
+                                [{ text: 'Volatility 25 Index'     }, { text: it.L('USD 1 per point') }, { text: five_tenth }, { text: hundredth }],
+                                [{ text: 'Volatility 50 Index'     }, { text: it.L('USD 1 per point') }, { text: '3.00'     }, { text: hundredth }],
+                                [{ text: 'Volatility 75 Index'     }, { text: it.L('USD 1 per point') }, { text: hundredth  }, { text: hundredth }],
+                                [{ text: 'Volatility 100 Index'    }, { text: it.L('USD 1 per point') }, { text: tenth      }, { text: hundredth }],
+                                [{ text: 'HF Volatility 10 Index'  }, { text: it.L('USD 1 per point') }, { text: five_tenth }, { text: hundredth }],
+                                [{ text: 'HF Volatility 50 Index'  }, { text: it.L('USD 1 per point') }, { text: '3.00'     }, { text: hundredth }],
+                                [{ text: 'HF Volatility 100 Index' }, { text: it.L('USD 1 per point') }, { text: '3.00'     }, { text: hundredth }],
```

Comment:
please use a variable for this too instead of repeating it

Reply:
Sure